doc/tools/make_rst.py

```diff
@@ -903,11 +904,12 @@ def rstize_text(text, state):  # type: (str, State) -> str
             break
 
         pre_text = text[:pos]
+        pos += 1
         indent_level = 0
-        while text[pos + 1] == "\t":
+        while pos < len(text) and text[pos] == "\t":
             pos += 1
             indent_level += 1
-        post_text = text[pos + 1 :]
+        post_text = text[pos:]
```

Comment:
Are you sure about these changes though? `pos` is used later in this method so it might break some assumptions (now it has different value after these lines).
Leaving these as `pos + 1` is probably safer (unless you're sure the rest of the logic is valid with these changes).

Reply:
You're right, the changes incorrectly handle the input having two consecutive line breaks.